src/components/shared/BackNextButtons/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+type BackNextButtonsProps = {
+  pageNum: number;
+  totalPages: number;
+  setPage: (func: any) => void;
+  onSubmit: (data: any) => void;
+};
+
+const BackNextButtons = ({
```

Comment:
I abstracted this component early on because we already have some prototypes for the upcoming business case. The biggest problem I can potentially see here is if we want to change the labels for these components for different pages/forms.